core/src/main/java/org/mobilitydata/gtfsvalidator/notice/MissingRecommendedFileNotice.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
You can add your organization to the copyright if you'd like and also update the year.

Reply:
If you change it, you can add `[acceptance test skip]` at the end of your commit message so the acceptance tests don't run once again.